go.mod

```diff
@@ -6,13 +6,15 @@ require (
 	cloud.google.com/go v0.47.0 // indirect
 	cloud.google.com/go/bigquery v1.2.0 // indirect
 	cloud.google.com/go/storage v1.2.1 // indirect
+	github.com/alecthomas/kingpin v2.2.6+incompatible
```

Comment:
Don't bump module versions unnecessarily - if they need to be bumped, it is normally better todo it in a new PR

Reply:
This new dependency was added automatically. I don't know where is it used.